backend/src/api/product-list/content-types/product-list/schema.json

```diff
@@ -241,6 +241,14 @@
          "relation": "manyToMany",
          "target": "api::faq.faq",
          "inversedBy": "product_lists"
+      },
+      "optionalFilters": {
```

Comment:
Sadly, we need to keep strapi pulls separate from the frontend pulls. This is because there are separate deploy processes. If the frontend is deployed first (which it usually is) then it will start asking for this field *before* it's actually available in strapi and strapi will respond with an error during the time between deployments.

It's possible this will only be for a few minutes, but it's also possible that it takes longer or that strapi deploy fails.